metadata-load/src/test/java/uk/ac/ebi/ampt2d/metadata/importer/database/SraObjectsImporterThroughDBTest.java

```diff
@@ -102,6 +104,23 @@ public void importAnalysisObject() throws Exception {
         assertEquals(1, referenceSequenceRepository.count());
     }
 
+    @Test
+    @Category(OracleDbCategory.class)
+    public void importAnalysisObjectWithReferenceAlignmentType() throws Exception {
+        Analysis analysis = sraObjectImporter.importAnalysis("ERZ000275");
+        assertNotNull(analysis);
+        assertEquals("ERZ000275", analysis.getAccessionVersionId().getAccession());
+        assertEquals(Analysis.Technology.UNSPECIFIED, analysis.getTechnology());
+        assertEquals(1, analysis.getFiles().size());
+        assertEquals("ERP001373", analysis.getStudy().getAccessionVersionId().getAccession());
+        List<ReferenceSequence> referenceSequences = analysis.getReferenceSequences();
+        referenceSequences.parallelStream().allMatch(referenceSequence -> referenceSequence.getType().equals
+                (ReferenceSequence.Type.GENE));
```

Comment:
Again, the problem with this is that ERZ000275 doesn't actually have any `ReferenceSequence.Type.GENE` sequences. It has:

``` xml
<ANALYSIS_TYPE>
    <REFERENCE_ALIGNMENT>
        <ASSEMBLY>
            <STANDARD refname="GRCh37-lite"/>
        </ASSEMBLY>
        <SEQUENCE accession="CM000663.1" label="1"/>
        <SEQUENCE accession="CM000664.1" label="2"/>
        <SEQUENCE accession="CM000665.1" label="3"/>
        [...]
    </REFERENCE_ALIGNMENT>
</ANALYSIS_TYPE>
```

These shouldn't be treated as accessioned sequences.

Reply:
In fact, I've queried EGA and wasn't able to find **a single example** of an analysis which would have either a transcriptome or an accessioned sequence. In `<SEQUENCE>` tags they have human chromosome accessions or some identifiers which don't resolve to anything. Maybe I'm wrong and EGA does have some transcriptomes/gene sequences (then can someone please provide examples?), otherwise we'll have to use examples from EVA.

Reply:
oh okay, I thought its an accessioned sequence :(. Not sure how to differentiate them though.

Reply:
https://github.com/EBIvariation/metadata-ws/pull/61#discussion_r302912994